lms/templates/dashboard.html

```diff
@@ -175,7 +176,7 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
         </li>
         % endif
 
-        <%include file='dashboard/_dashboard_status_verification.html' />
+        <%include file="${microsite.get_template_path('dashboard/_dashboard_status_verification.html')}" />
```

Comment:
Note to reviewers, this is how to allow for microsites to override the included template. If there is no override, then the template in edx-platform is used.